### PR TITLE
Adds a new, terrible kind of tomato.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds disgusting tomatoes, which contain what is basically spewium-lite (no organ vomiting). 

## Why It's Good For The Game

Fun.

## Changelog
:cl:
add: disgusting tomatoes
/:cl:

### CODE REVIEW

**Comment by TheSpaghetti** [comment]
Trying to add the chemical that makes the tomatoes work, don't merge this PR yet.

File: code/modules/hydroponics/grown/tomato.dm [hunk 1]
```diff
@@ -145,3 +145,31 @@
...
+	seed = /obj/item/seeds/tomato/disgusting
+	name = "disgusting tomato"
+	desc = "It smells awful."
+	icon_state = "tomato"
+	splat_type = /obj/effect/decal/cleanable/vomit
+	filling_color = "#FF6347"
+	bitesize_mod = 2
+	foodtype = FRUIT, GROSS
+	grind_results = list(/datum/reagent/toxin/disgustingjuice = 0)
+	juice_results = list(/datum/reagent/toxin/disgustingjuice = 0)
```
Comment (Sneakyrat6) [inline code/modules/hydroponics/grown/tomato.dm:174]: 0 units?

**Comment by timothyteakettle** [comment]
vomiting has no real use outside of forcing people to be stunlocked

the moment cycle 10 passes they're literally stuck forever until the chemical exits their system

also just bloats botany plants with more powercreep which just isn't what is needed right now

**Comment by ghost** [comment]
> vomiting has no real use outside of forcing people to be stunlocked
> 
> the moment cycle 10 passes they're literally stuck forever until the chemical exits their system
> 
> also just bloats botany plants with more powercreep which just isn't what is needed right now

counterpoint: memes

File: code/modules/hydroponics/grown/tomato.dm (hunk 1 above)
Comment (ghost) [reply code/modules/hydroponics/grown/tomato.dm:174]: The numbers were like that on regular tomatoes.

**Comment by timothyteakettle** [comment]
memes is not a reason to create a balance issue

**Comment by silicons** [comment]
the teapot is correct

**Pull request closed without merging**